### PR TITLE
[docs] Deckhouse network security requirements

### PR DESCRIPTION
**Pull request by apolovov**
## Description
Summary table of necessary ports using by Deckhouse.

## Why do we need it, and what problem does it solve?
In some environments it is a rule to configure network policies in infrastructure. The table describes the used ports.

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: docs
type: chore
summary: Summary table of necessary ports using by Deckhouse.
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: docs/documentation/pages/NETWORK_SECURITY_SETUP_RU.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+---
+title: Настройка сетевых политик для работы Deckhouse
+permalink: ru/network_security_setup.html
+lang: ru
+---
+
+Если в инфраструктуре, где работает Deckhouse, есть требования для ограничения сетевого взаимодействия, то необходимо соблюсти следующие условия:
+
+* Включен [режим туннелирования](modules/021-cni-cilium/configuration.html#parameters-tunnelmode) трафика между подами.
+* Разрешено взаимодействие между узлами по следующим портам:
```
Comment (z9r5) [inline docs/documentation/pages/NETWORK_SECURITY_SETUP_RU.md:10]:
```suggestion
Если в инфраструктуре, где работает Deckhouse Kubernetes Pllatform, есть требования для ограничения сетевого взаимодействия, то необходимо соблюсти следующие условия:

* Включить [режим туннелирования](modules/021-cni-cilium/configuration.html#parameters-tunnelmode) трафика между подами.
* Разрешить взаимодействие между узлами по портам, приведенным в таблицах на текущей странице.
```

File: docs/documentation/pages/NETWORK_SECURITY_SETUP.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
+---
+title: Configuring network policies for Deckhouse
+permalink: en/network_security_setup.html
+lang: en
+---
+
+If the infrastructure where Deckhouse is running has requirements to limit network communication, the following conditions must be met:
+
+* [Tunneling mode](modules/021-cni-cilium/configuration.html#parameters-tunnelmode) for traffic between pods is enabled.
+* Inter-node communication is allowed on the following ports:
```
Comment (z9r5) [inline docs/documentation/pages/NETWORK_SECURITY_SETUP.md:10]:
```suggestion
If the infrastructure where Deckhouse Kubernetes Pllatform is running has requirements to limit network communication, the following conditions must be met:

* [Tunneling mode](modules/021-cni-cilium/configuration.html#parameters-tunnelmode) for traffic between pods is enabled.
* Inter-node communication is allowed on the ports shown in the tables on the current page.
```

**Review by z9r5 — APPROVED** [review]

File: docs/documentation/pages/NETWORK_SECURITY_SETUP_RU.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+---
+title: Настройка сетевых политик для работы Deckhouse
+permalink: ru/network_security_setup.html
+lang: ru
+---
+
+Если в инфраструктуре, где работает Deckhouse Kubernetes Pllatform, есть требования для ограничения сетевого взаимодействия, то необходимо соблюсти следующие условия:
```
Comment (apolovov) [inline docs/documentation/pages/NETWORK_SECURITY_SETUP_RU.md:7]:
```suggestion
Если в инфраструктуре, где работает Deckhouse Kubernetes Platform, есть требования для ограничения сетевого взаимодействия, то необходимо соблюсти следующие условия:
```

File: docs/documentation/pages/NETWORK_SECURITY_SETUP.md [hunk 4]
```diff
@@ -0,0 +1,14 @@
+---
+title: Configuring network policies for Deckhouse
+permalink: en/network_security_setup.html
+lang: en
+---
+
+If the infrastructure where Deckhouse Kubernetes Pllatform is running has requirements to limit network communication, the following conditions must be met:
```
Comment (apolovov) [inline docs/documentation/pages/NETWORK_SECURITY_SETUP.md:7]:
```suggestion
If the infrastructure where Deckhouse Kubernetes Platform is running has requirements to limit network communication, the following conditions must be met:
```

**Review by qw1mb0 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @apolovov. I've started the workflow for label 'deploy/web/stage'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

File: docs/documentation/_data/deckhouse-ports.yml [hunk 5]
```diff
@@ -0,0 +1,256 @@
...
+  - ports: "3367"
+    protocol: TCP
+    description:
+      en: API of the *sds-replicated-volume* module node agent
+      ru: API агента модуля *sds-replicated-volume*
+  - ports: "9942"
+    protocol: TCP
+    description:
+      en: sds-replicated-volume node agent metrics
+      ru: Метрики агента sds-replicated-volume
```
Comment (z9r5) [inline docs/documentation/_data/deckhouse-ports.yml:163]:
```suggestion
      en: "*sds-replicated-volume* node agent metrics"
      ru: "Метрики агента *sds-replicated-volume*"
```

File: docs/documentation/_data/deckhouse-ports.yml [hunk 6]
```diff
@@ -0,0 +1,256 @@
...
+  - ports: "7000-7999"
+    protocol: TCP
+    description:
+      en: "*sds-replicated-volume* DRBD replication"
+      ru: "Репликация DRBD для *sds-replicated-volume*"
+  - ports: "49152, 49153"
+    protocol: TCP
+    description:
+      en: "*Deckhouse Virtualization Platform* VM live migration port"
+      ru: "Живая миграция ВМ в *Deckhouse Virtualization Platform*"
```
Comment (z9r5) [inline docs/documentation/_data/deckhouse-ports.yml:173]:
```suggestion
      en: "Deckhouse Virtualization Platform VM live migration port"
      ru: "Живая миграция ВМ в Deckhouse Virtualization Platform"
```

**Review by z9r5 — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Review by konstantin-axenov — APPROVED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Cherry pick PR [8182](https://github.com/deckhouse/deckhouse/pull/8182) to the branch [release-1.59](https://github.com/deckhouse/deckhouse/tree/release-1.59) successful!

**Pull request merged**